src/applications/edu-benefits/0993/tests/e2e/00.0993-opt-out.cypress.spec.js

```diff
@@ -0,0 +1,58 @@
+import Timeouts from 'platform/testing/e2e/timeouts';
+import applicationSubmit from './fixtures/mocks/application-submit.json';
+import testData from './fixtures/test-data.json';
+
+describe('Opt Out Test', () => {
+  it('Fills out the form and submits', () => {
+    cy.intercept(
+      'POST',
+      '/v0/education_benefits_claims/0993',
+      applicationSubmit,
+    ).as('optOutClaim');
+
+    cy.visit('/education/opt-out-information-sharing/opt-out-form-0993/');
+    cy.get('body').should('be.visible');
+
+    // Claimant information
+    cy.get('input[name="root_claimantFullName_first"]', {
+      timeout: Timeouts.normal,
+    });
+    cy.injectAxeThenAxeCheck();
+
+    cy.fillName('root_claimantFullName', testData.claimantFullName);
+    cy.get('input[name="root_view:noSSN"]').click();
+    cy.get('input[name="root_vaFileNumber"]', {
+      timeout: Timeouts.slow,
+    }).click();
+    cy.fill('input[name="root_vaFileNumber"]', testData.vaFileNumber);
+    cy.get('.form-progress-buttons .usa-button-primary').click();
+
+    // Review and submit page
+    cy.url({ timeout: Timeouts.slow }).should(
+      'not.contain',
+      '/claimant-information',
+    );
+
+    cy.get('label[name="privacyAgreementAccepted-label"]', {
+      timeout: Timeouts.slow,
+    })
+      .should('be.visible')
+      .then(() => {
+        cy.wait(Timeouts.normal);
```

Comment:
Can we use something other than `cy.wait()` here?

Reply:
Replaced this with adding the timeout to the `cy.get` on the following line instead